react/features/participants-pane/components/web/FooterContextMenu.tsx

```diff
@@ -147,6 +151,9 @@ export const FooterContextMenu = ({ isOpen, onDrawerClose, onMouseLeave }: IProp
                     onClick: muteAllVideo,
                     text: t('participantsPane.actions.stopEveryonesVideo')
                 } ] } />
+            {raisedHandsQueue.length !== 0 && (<LowerHandButton
```

Comment:
Is this check correct also when it's just the local user the one who raised the hand?

Reply:
Hi Saghul, I fixed this part by adding a condition `if (!participantID) {
                dispatch(raiseHand(false));
            }` in the LowerHandButton.tsx.